sensorlib/doc/CarlaCDASimAPI.md

```diff
@@ -0,0 +1,208 @@
+# Carla CDA Sim API
+
+The Carla CDA Sim API provides programmatic access to the library functionality. The primary means of
+accessing features is through a CarlaCDASimAPI object, which may be constructed from any piece of information sufficient to
+establish a Carla connection.
+
+First ensure the following environment variables are set which define the CARLA version being utilized:
+
+| Environment Variable | Description                                                                                                       |
+| -------------------- |-------------------------------------------------------------------------------------------------------------------|
+| LOAD_CARLA_EGG       | Set to True to load the CARLA Python API from an egg file. Leave blank to use a pip-managed CARLA client library. |
+| CARLA_VERSION        | The CARLA version to load (ex: "0.0.14").                                                                         |
+| CARLA_EGG_DIR        | (optional) Hint directory for searching for the CARLA egg file.                                                   |
+
+
+Nex, build the API object.
+
+```
+api = CarlaCDASimAPI.build_from_host_spec(carla_host, carla_port)
+
+# or...
+
+api = CarlaCDASimAPI.build_from_client(carla_client)
+
+# or...
+
+api = CarlaCDASimAPI.build_from_world(carla_world)
+```
+
+The API object may then be used to indirectly build and manage sensors, and request output data. Sensors are located by
+association with an infrastructure item, such as a traffic light post, and the sensor index within a co-located
+collection of sensors. For example, a sensor with an infrastructure_id of 3 and sensor_id 0, is the 0th sensor
```

Comment:
Lets create a future issue to rename `infrastructure_id` to something else. It conflicts with CDASim's definition of infrastructure_id. Not terribly important but we want some thing to track this. Once we move the code somewhere. I will try to add a github issue to address this.

Reply:
That's understandable if there is a difference between this concept and cda infrastructure ID. Are they not the same?